commands/upgrade.js

```diff
@@ -66,8 +66,8 @@ const upgrade /*: Upgrade */ = async ({root, args}) => {
 
 const update = (meta, type, name, version, from) => {
   if (meta[type] && meta[type][name]) {
-    const min = minVersion(meta[type][name]);
```

Comment:
this would throw if upgrading and the existing version is not valid semver, e.g. `workspace:*`